CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is always welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md). We expect that all project participants will adhere to it, and any actions taken that go against the Code of Conduct will not be tolerated.
+
+These guidelines are an attempt at better addressing the incoming issues and pull requests. 
+Please read them closely, and feel free to ask any questions if needed.
+
+## Getting started
+### Submitting Pull Requests
+
+You might know the basics of 
+[contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork
+[properly set up](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
```

Comment:
I think we have two options here.

You can either do this:
```suggestion
You might know the basics of [contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork [properly set up](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
```

Or you can do this:
```suggestion
Before submitting a pull request (PR), please ensure you are familiar with the following: 
- [Contributing on GitHub](https://help.github.com/articles/using-pull-requests)
- [Properly setting up a fork](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
```

I personally prefer the latter, but I'm open to discuss it.

Reply:
I'd agree to the latter as well